src/main/resources/public/css/stylesheet.css

```diff
@@ -55,7 +45,35 @@ span {
     margin-right: 2em;
 }
 
-html, body {
-    height:100vh;
+html,
+body {
+    height: 100vh;
     color: #3b3d44;
-}
\ No newline at end of file
+}
+
+input, textarea {
+    width: 100%;
+    margin: 8px 0;
+}
+
+input[type=text], textarea{
```

Comment:
did you make sure these don't effect the styling of our other inputs on the website? since these are very generic but have quite a few rules I want to make sure there's no bad side effects

Reply:
the only other input we have is the search bar but the styling is overriden by the bootstrap theme that it's part of since they have specific rules.

Reply:
actually, it seems the jquery table we are using doesn't override the css rules like the search bar.

Reply:
i'll put the the forms in a div and apply the rules to those divs

Reply:
input[type=text], textarea{ => should have a space: input[type=text], textarea {

these aren't following the single selector per line as your linter does